### DISCUSSION TITLE
How to bind a key to a menu item

### DISCUSSION THREAD

**Discussion by paaguti** [orbitalquark/textadept#326] [Q&A]
Hi,

time permitting, it would be nice to include a sample of how one could bind a key to a menu entry.
A good example would be:

How can I bind `'ctrl+,'` to Edit -> Preferences ?

Thx, /PA

**Comment by orbitalquark** [comment]
Sure: https://github.com/orbitalquark/textadept/commit/ec291609db76260dad168c951e93faf269ee7cd7
